.werft/observability/monitoring-satellite.ts

```diff
@@ -68,7 +66,7 @@ export class MonitoringSatelliteInstaller {
                 domain: '${previewDomain}',
                 nodeExporterPort: ${nodeExporterPort},
             },
-            ${withVM ? "" : "nodeAffinity: { nodeSelector: { 'gitpod.io/workload_services': 'true' }, },"}
+            ${"nodeAffinity: { nodeSelector: { 'gitpod.io/workload_services': 'true' }, },"}
```

Comment:
this line should be removed, shouldn't it? The code looks like it was only executed for `withVM == false` before.